ui/components/connections/index.js

```diff
@@ -263,9 +276,10 @@ function Connections(props) {
       .unwrap()
       .then((resp) => {
         notify({
-          message: `Environment: ${resp.Name} saved`,
+          message: `Environment: ${resp.name} saved`,
```

Comment:
```suggestion
          message: `Environment "${resp.name}" updated`,
```

Reply:
This message is for creating the env (reference: https://github.com/meshery/meshery/pull/9831/files#diff-758de2b85e43b78da1cb737ebe181174f2825226d111eff469ded13847a4e062R269), so I believe we should have `saved`. Though we can update to `created` if that sounds more appropriate.

Reply:
yes, created sounds more appropriate i guess